.github/workflows/django-multitenant-tests.yml

```diff
@@ -0,0 +1,80 @@
+on:
+  push:
+    branches:
+      - "**"
+  workflow_dispatch:
+jobs:
+  static-checks:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+    - uses: actions/checkout@v3
+    - name: Install python dependencies
+      run: |
+        pip install -r requirements/static-analysis-requirements.txt
```

Comment:
What if the dependencies in there were added to `requirements/test-requirements.txt`, and then you could run `make dev-dependencies`? Seems like static analysis is part of the testing process.

Reply:
I want to separate them since they have totally different dependency structure
For example, we don't have Django in test dependencies, since we install it separately for each Django version, but we perform static analysis tests using latest Django version

We don't need black, prospector for test 
In summary, to keep the actual test dependencies clean, I want to keep them separate